go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/evanphx/json-patch/v5 v5.6.0 // indirect
 	github.com/fsnotify/fsnotify v1.6.0 // indirect
 	github.com/go-logr/logr v1.2.4 // indirect
+	github.com/ghodss/yaml v1.0.0 // indirect
```

Comment:
do we need it or can just use `gopkg.in/yaml`?

Reply:
This package is needed it seems , because with gopkg.in/yaml i get error on unmarshaling like `cannot unmarshal !!str `64Mi` into resource.Quantity`

Reply:
Behind the scenes the package does use gopkg.in/yaml but converts the yaml to json and then json.unmarshal

Reply:
ok. I'd say we can use `https://github.com/kubernetes-sigs/yaml` as [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes/blob/15a1f9a39db95a7f83ce5463a328566411d99d51/go.mod#LL128C2-L128C18). It's a permanent fork of github.com/ghodss/yaml